tests/test_Alias.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import pytest
+import xml.etree.ElementTree as ET
+import xml_common
```

Comment:
Please do not import anything from the test directory.

Move common code to `python/lsst/ts/xml/`. I suggest a module name of `test_utils.py` because we use that a lot for this kind of thing. Naming the module `xml_common` would not be appropriate because all ts_xml library code is, by definition, common to all of the ts_xml package.

This allows the tests to be run from any directory and also makes the code easier to follow, in that common code is in the standard location for library modules.

Reply:
Moved xml_common.py in tests/ to python/lsst/ts/xml/test_utils.py.